src/jump_input.jl

```diff
@@ -0,0 +1,139 @@
+using BilevelJuMP
+using Ipopt
+using JuMP
+using Test
+
+model = BilevelModel(Ipopt.Optimizer, mode = BilevelJuMP.ProductMode(1e-9))
+
+@variable(Upper(model), y, start = 8 / 15)
+@variable(Lower(model), x, start = 3.5 * 8 / 15)
+@objective(Upper(model), Min, 3x + y)
+@constraints(Upper(model), begin
+    u1, x <= 5
+    u2, y <= 8
+    u3, y >= 0
+end)
+
+@objective(Lower(model), Min, -x)
+@constraint(Lower(model), l1,  x +  y <= 8)
+@constraint(Lower(model), l2, 4x +  y >= 8)
+@constraint(Lower(model), l3, 2x +  y <= 13)
+@constraint(Lower(model), l4, 2x - 7y <= 0)
+
+function _build_single_model(
+    model::BilevelModel,
+)
+    upper = JuMP.backend(model.upper)
+    lower = JuMP.backend(model.lower)
+    linkUL = JuMP.index(model.upper_to_lower_link)
+    linkLU = JuMP.index(model.lower_to_upper_link)
+
+    return _build_single_model(upper, lower, linkUL, linkLU)
+end 
+
+function _build_single_model(
+    upper::MOI.ModelLike, 
+    lower::MOI.ModelLike, 
+    # A dictionary that maps variables in the upper to variables in the lower
+    linkUL::Dict{MOI.VariableIndex,MOI.VariableIndex},
+    linkLU::Dict{MOI.VariableIndex,MOI.VariableIndex}
+)
+    # A new model to build
+    model = MOI.Utilities.Model{Float64}()
```

Comment:
```suggestion
    model = MOI.FileFormats.MPS.Model()
```

We may as well build the MPS model directly